### ISSUE TITLE
Export Failed for an unknown reason

### ISSUE THREAD

**Issue by Writinglazy** [mbloch/mapshaper#293]
Hey!

I downloaded all Guesthouses in Germany via the overpass Interface as geojson and now tried to convert them to a shapefiled. I get the error: Export failed for an unknown reason.

When i did exactly the same with all the hotels it worked out fine. Any suggestions what could be the reason?
The geojson file is about 9 Mb large. Shouldn´t be to large, right?

Thanks for your awesome tool!

[Guesthouse_GER.zip](https://github.com/mbloch/mapshaper/files/2256489/Guesthouse_GER.zip)

**Comment by Writinglazy** [comment]
I just realized that the same error comes up wenn i try to convert all the Guesthouses in Brandenburg (which is a small federal state of germany and contains even less data). So is doesn´t seem to be about the size...

**Comment by mbloch** [comment]
Hi, thanks for reporting this :)

The error was caused by non-ascii characters in some of the field names. I just published a new mapshaper version that converts non-alphanumeric characters are now converted to underscores in field names.

Shapefile field names are limited to 10 characters in length, so most of the field names from your original GeoJSON file are getting truncated.

If you retry your GeoJSON file, you'll notice that three Shapefiles are getting created... this is because the GeoJSON file has polygon, line and point features, whereas Shapefiles are limited to a single geometry type.
